roles/openshift_grafana/tasks/gf-permissions.yml

```diff
@@ -1,25 +0,0 @@
----
-- name: Create grafana user on htpasswd
```

Comment:
Can you explain why you removed this role ?!

Reply:
So, whats gonna be if it's not specified, what the defaults ?
we've added this role provide limited access only the grafana UI for certain users.
otherwise it means we'll need to use the oc token or user use \pass which has cluster privileged

Reply:
But this means you overwrite any other identity provider settings of the cluster. So if I have ldap in my cluster - it gets deleted. We should do as you say - use openshift rbac for this and adjust rolebindings for access. I think you need just access to grafana namespace. If not - proxy might be configured wrong

Reply:
I just checked. My cluster has user `admin`. So if I do `oc policy add-role-to-user view admin -n grafana` all good and the user can see dashboards. This should suffice user auth concerns :)

Reply:
Ok, sounds good approach.
but seems  like we need to update the PR with this policy

Reply:
The thing is we dont know what users playbooks consumer will have. So we can commit to the name of the roles. This should be part of readme at best, and maybe separate step where based on env variable role binding would be created.